ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.15.yaml

```diff
@@ -320,7 +320,7 @@ tests:
 - as: ocp-e2e-ibmcloud-ovn-multi-ppc64le
   cron: 20 7,17 * * *
   steps:
-    cluster_profile: ibmcloud-multi-ppc64le
```

Comment:
this is the existing mac job... please don't change.

Reply:
Done, removed changes from 4.15 jobs